readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: ['htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia']
+formats: list(enum('htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia'), required=False)
+
+# The path to the requirements file from the root of the project
+requirements_file: str(required=False)
+
+# Configuration for Conda support
+conda: include('conda', required=False)
+
+# Configuration for the documentation build process
+build: include('build', required=False)
+
+# Configuration of the Python executable to be used
+python: include('python', required=False)
+
+# Configuration for sphinx
+sphinx: include('sphinx', required=False)
```

Comment:
Instead of the `Documentation type` I have added a `sphinx` key. We want to still support mkdocs, right? If so, do you prefer the `documentation type` key back or having a `mkdocs`?

Reply:
We could use this to extend the build and support things like https://github.com/rtfd/readthedocs.org/issues/1139

Reply:
Not sure. It does seem reasonable to have a `sphinx` and `mkdocs` section, and just always default to Sphinx unless we see a `mkdocs` section or something.

Reply:
Having two different sections could be useful be extended in the future.

In case this works as Eric mentioned, we will need a check to to accept one or the other, but not both.